src/scaladoc/scala/tools/nsc/doc/html/SyntaxHigh.scala

```diff
@@ -41,15 +41,15 @@ private[html] object SyntaxHigh {
     "varargs", "volatile")
 
   /** Standard library classes/objects, sorted alphabetically */
-  val standards = Array (
-    "WeakTypeTag", "Any", "AnyRef", "AnyVal", "App", "Array",
-    "Boolean", "Byte", "Char", "Class", "ClassTag", "ClassManifest",
+  val standards = Array(
+    "Any", "AnyRef", "AnyVal", "App", "Array",
+    "Boolean", "Byte", "Char", "Class", "ClassManifest", "ClassTag",
     "Console", "Double", "Enumeration", "Float", "Function", "Int",
     "List", "Long", "Manifest", "Map",
     "NoManifest", "None", "Nothing", "Null", "Object", "Option", "OptManifest",
     "Pair", "Predef",
     "Seq", "Set", "Short", "Some", "String", "Symbol",
-    "Triple", "TypeTag", "Unit")
+    "Triple", "TypeTag", "Unit", "WeakTypeTag")
```

Comment:
Avoid this issue recurring by codifying it,

   val standards = Array(...
   ...
   ).sorted